WireAnalytics/Sources/WireAnalytics/TeamInfo.swift

```diff
@@ -27,9 +27,9 @@ public struct TeamInfo {
     public let role: String
 
     /// The size of the team the user belongs to.
-    public let size: Int
+    public let size: UInt
```

Comment:
I get your point that a negative count value doesn't make sense, but wouldn't it be better to match the type of the property we want to track, like `conversation.localParticipants.count`?